pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -135,57 +146,32 @@ func defaultStrategyChooser(instance *v1alpha1.Jaeger) strategy.S {
 	return strategy.For(context.Background(), instance)
 }
 
-func (r *ReconcileJaeger) handleCreate(str strategy.S) (bool, error) {
-	objs := str.Create()
-	created := false
-	for _, obj := range objs {
-		err := r.client.Create(context.Background(), obj)
-		if err != nil && !apierrors.IsAlreadyExists(err) {
-			log.WithError(err).Error("failed to create")
-			return false, err
-		}
-
-		if err == nil {
-			created = true
-		}
-	}
-
-	return created, nil
-}
-
-func (r *ReconcileJaeger) handleUpdate(str strategy.S) error {
-	objs := str.Update()
-	for _, obj := range objs {
-		if err := r.client.Update(context.Background(), obj); err != nil {
-			log.WithError(err).Error("failed to update")
-			return err
-		}
-	}
-
-	return nil
-}
-
 func (r *ReconcileJaeger) handleDependencies(str strategy.S) error {
 	for _, dep := range str.Dependencies() {
 		err := r.client.Create(context.Background(), &dep)
 		if err != nil && !apierrors.IsAlreadyExists(err) {
-			log.WithError(err).Error("failed to create")
 			return err
 		}
 
-		// we probably want to add a couple of seconds to this deadline, but for now, this should be sufficient
-		deadline := time.Duration(*dep.Spec.ActiveDeadlineSeconds)
-		return wait.Poll(time.Second, deadline*time.Second, func() (done bool, err error) {
+		// default to 2 minutes, in case we get a null pointer
```

Comment:
can you elaborate on the "in case we get a null pointer" comment?

Reply:
`dep.Spec.ActiveDeadlineSeconds` is null if it's not specified. So far, only the Cassandra Create Schema job is a "dependency" and we control its deployment, so, it's always set, but I created a test without specifying this value and I got a null pointer panic.